server/api/api/endpoints/functions.py

```diff
@@ -798,14 +798,16 @@ def _deploy_nuclio_runtime(
             )
         if monitoring_application:
             monitoring_deploy = (
-                server.api.crud.model_monitoring.deployment.MonitoringDeployment()
+                server.api.crud.model_monitoring.deployment.MonitoringDeployment(
+                    project=fn.metadata.project,
+                    auth_info=auth_info,
+                    model_monitoring_access_key=model_monitoring_access_key,
+                    db_session=db_session,
```

Comment:
Let's keep the same parameter order every time.

Reply:
Done in 441f0a5.